src/main/resources/db/migration/V3_1__drop_appuser_name_col.sql

```diff
@@ -1 +1 @@
-ALTER TABLE app_user DROP COLUMN IF EXISTS "name";
\ No newline at end of file
+ALTER TABLE IF EXISTS app_user DROP COLUMN IF EXISTS "name";
```

Comment:
we should leave out this change to the migration, since it was just added for local dev and shouldn't be an issue with the separate profiles implemented
```suggestion
ALTER TABLE app_user DROP COLUMN IF EXISTS "name";
```